src/net/sourceforge/plantuml/tim/stdlib/Modulo.java

```diff
@@ -30,6 +30,7 @@
  *
  *
  * Original Author:  Arnaud Roques
+ * Contribution:  Aravind Pai
```

Comment:
I didn't realize I could do this in my previous PR, so I'd like to squeeze it in with this one. 😄